library/ix-dev/community/netbootxyz/values.yaml

```diff
@@ -0,0 +1,30 @@
+image:
+  repository: netbootxyz/netbootxyz
+  pullPolicy: IfNotPresent
+  tag: 0.7.0-nbxyz2
+
+resources:
+  limits:
+    cpu: 4000m
+    memory: 8Gi
+
+netbootConfig:
+  additionalEnvs: []
+
+netbootNetwork:
+  tftpPort: 30031
+  webHttpPort: 30032
+  webAssetsPort: 30033
+  # TFTP only works with hostNetwork: true
+  hostNetwork: true
```

Comment:
Quick look in the upstream container, shows that the container binds port `80` for the nginx webserver.
Setting hostnetwork to true will break access to truenas webui or at the very least conflict.

We have to override the nginx config and dynamically update the port based on the port provided by the user